.github/workflows/build.yml

```diff
@@ -25,8 +25,12 @@ jobs:
       - name: Install dependencies and package
         shell: bash
         run: pip install -U -e .'[tests]'
-      - name: Run docstring tests
-        run: pytest --doctest-modules --ignore=pyxem/tests pyxem
+      - name: TEMP Install development version of HyperSpy, only for testing!
+        shell: bash
+        run: pip install https://github.com/hyperspy/hyperspy/archive/refs/heads/RELEASE_next_minor.tar.gz
+      - name: TEMP Install older skimage version
+        shell: bash
+        run: pip install scikit-image==0.18.3
```

Comment:
I think this is unnecessary if #790 is merged.

Reply:
As in tests will run if that PR is merged, but more of the ones listed above will fail because `skimage.draw.circle()` isn't available. But that's unrelated to this PR, so should be OK.

Reply:
Thanks, @pc494.